ui/vic-ui-h5c/vic/src/vic-app/src/app/services/vm-view.service.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('VicVchViewService', () => {
             sorting: 'id,asc'
         });
 
-        service.vchs$.subscribe(vchs => {
+        service.reloadVchs({}).subscribe(vchs => {
```

Comment:
service method should only fetch the data not reload so make the name reflect that here and below.